test/core/gear_log/file_handle_test.exs

```diff
@@ -46,6 +46,16 @@ defmodule AntikytheraCore.GearLog.FileHandleTest do
     assert_file_content([log_msg])
   end
 
+  test "write/2 should accept malformed (non-UTF8) log message" do
+    log_with_malformed_msg = {@log_time, :info, @context_id, <<222>>}
```

Comment:
It'd be nicer if we change `<<222>>` to something like `:crypto.strong_rand_bytes(10)`.

Reply:
fixed in b26931b